manifests/config/sample_rulesets.pp

```diff
@@ -0,0 +1,57 @@
+# NOTE: THIS IS A [PRIVATE](https://github.com/puppetlabs/puppetlabs-stdlib#assert_private) CLASS**
+#
+# @summary This class is called from auditd for service config.
+#
+# @author https://github.com/simp/pupmod-simp-auditd/graphs/contributors
+#
+class auditd::config::sample_rulesets {
+  $_sample_rules_basedir = '/usr/share/audit/sample-rules'
+  # At this point we should not have an empty rulesets list...so we need to
+  # validate the list; we should be OK skipping nonexistent ones those, but
+  # SHOULD notify of this
+  $auditd::sample_rulesets.each |$_ruleset| {
+    if $_ruleset in keys($facts['auditd_sample_rulesets']) {
+      $_order = $facts['auditd_sample_rulesets'][$_ruleset]['order']
+
+      # Copy the ruleset into place via puppet file resource and notify service
+      if $_ruleset == 'privileged' {
```

Comment:
This is a good idea but I think we might want to split it out into something individual that can be optionally included.

The main reason is that the `filecap` command is not present on a minimized system by default but that also means that we don't quite know what may end up in the scripts over time.

Reply:
Yeah, I mean this is entirely optional as is...if someone does not include 'privileged' in the array of sample_rulesets to include, it will not process. I was jammed up on the premise that we can't just stub this out as a template because we'd be constantly monitoring package releases for changes and we don't really have the notion of a common 'cache' module (which I'm starting to think may be a good idea).

Reply:
The exec that generates the `*.evaluated` file shoudl fail if the commands are not present as well...

Reply:
@ayohrling It is pretty scope limited so it's probably fine. Are you OK marking it as experimental here and in the README so that we don't cause a ruckus if we need to break it?